topology/probes/k8s/container.go

```diff
@@ -86,16 +86,35 @@ func containerUID(pod *v1.Pod, containerName string) graph.Identifier {
 	return graph.GenIDNameBased(string(pod.GetUID()), containerName)
 }
 
+func dumpContainer2(podNamespace, podName, containerName string) string {
```

Comment:
`func dumpContainer(pod *v1.Pod, containerName string) string` may be enough

Reply:
I wanted to have a dump for containers when we don't have readily available kubernetes objects (such as v1.Pod, etc.)

Reply:
For the only call to dumpContainer2 (line 175 of container.go), we have v1.Pod object

Reply:
ok, will be addressed in: `k8s: improve logging of CRUD operations`